random_data.go

```diff
@@ -0,0 +1,178 @@
+//
+// Copyright:: Copyright 2017 Chef Software, Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package main
+
+// Random data to use by the generator.go file to create random nodes/ccr's
```

Comment:
I really need help shaping this file, if we want something to be displayed in the UI, this is the file we have to modify the Raw Data. You can push changes or post ideas and I will commit them. (either way, thank you!!!)